mlflow/legacy_databricks_cli/configure/provider.py

```diff
@@ -204,6 +208,7 @@ def __init__(self):
             SparkTaskContextConfigProvider(),
```

Comment:
Can we please add a comment here that the precendece order of getting tokens is defined by the ordering of this list? 

It was very confusing to me on how the ordering was getting defined